scripts/deploy-graphiql.sh

```diff
@@ -45,4 +39,17 @@ aws s3 cp dist/ s3://graphiql.medplum.com/ \
   --exclude "*" \
   --include "*.txt"
 
+# Now deploy named files that are not cached.
+# These are small lightweight files that are not hashed.
+# It is important to deploy these files last,
+# because they reference the previously uploaded hashed files.
```

Comment:
🙏

Reply:
Is the rhyming of the first 3 lines here intentional? Felt like I was reading a clever rhyme written by ChatGPT or something